provisioners/ansible/playbooks/apps/create-aws-resources.yaml

```diff
@@ -14,6 +28,7 @@
         disable_rollback: true
         template: "../../../../templates/cloudformation/apps/aws-resources.yaml"
         template_parameters:
+          AWSELBAccountID: "{{ AWSELBAccountID.id }}"
```

Comment:
Why do we need `AWSELBAccountID.id` here? Why not just use the variable that's already available from `conf/ansible/inventory/group_vars/all.yaml` ?

Reply:
As ansible only retrieves variables once and the region has to be injected as part of the variable name it cannot be retrieved this way.

Reply:
It's possible. Here's an example https://github.com/shinesolutions/aem-aws-stack-builder/blob/master/provisioners/ansible/playbooks/apps/aem/full-set/main.yaml#L77 . `os_type` on that example is 'injected'.